bpf/process/bpf_execve_event.c

```diff
@@ -135,7 +135,8 @@ event_filename_builder(void *ctx, struct msg_process *curr, __u32 curr_pid,
 	if (!heap)
 		return bin;
 
-	probe_read_str(heap->pathname, 255, filename);
+	memset(heap->pathname, 0, 256);
```

Comment:
Usually it is a way better to have const here rather than hardconding values and losing the track... can be updated in next PR

Reply:
I added a commit to make that a define, thanks!